ui/DesignSystem/Primitive/Input/Password.svelte

```diff
@@ -0,0 +1,127 @@
+<script>
```

Comment:
```suggestion
<script lang="ts">
```

Now that our tooling works for tsed components lets always use it for new ones.